crates/core/c8y_api/src/smartrest/smartrest_deserializer.rs

```diff
@@ -306,6 +306,24 @@ impl SmartRestJwtResponse {
     }
 }
 
+#[derive(Debug, Deserialize, Serialize, Eq, PartialEq)]
+pub struct AvailableChildDevices {
+    pub message_id: String,
+    #[serde(default)]
+    pub devices: std::collections::HashSet<String>,
+}
+
+impl SmartRestRequestGeneric for AvailableChildDevices {}
```

Comment:
Is this empty implementation required?

Reply:
This is needed to get the `from_smartrest` trait method